src/Resources/ResourceFactory.php

```diff
@@ -195,6 +196,54 @@ protected function serviceFactory(array &$properties)
             unset($properties['attributes']['transit_time']['max']);
         }
 
+        if (isset($properties['attributes']['regions_from'][0])) {
+            $regionFromAttribute = $properties['attributes']['regions_from'][0];
+
+            $countryCode = $regionFromAttribute['country_code'];
+            $regionCode = $regionFromAttribute['region_code'];
+
+            /** @var RegionInterface $regionFrom */
+            $regionFrom = $this->api->getRegions([
+                    'country_code' => $countryCode,
+                    'region_code'  => $regionCode,
+                ]
```

Comment:
The closing method paranthesis `)` can be on the same line as the closing array bracket.

```suggestion
                ])->get()[0];
```